chapter2/tests/t19/terraform/_modules/container/main.tf

```diff
@@ -0,0 +1,21 @@
+resource "docker_image" "container_name" {
+  name         = var.container_image
+  keep_locally = var.container_image_keep_locally
+}
+
+resource "docker_container" "container_name" {
+  image = docker_image.container_name.image_id
+  name  = var.container_name
+  ports {
```

Comment:
> Please implement ports in the same dynamic way as `volumes`, because it is allowed to add as many ports as we want to docker container usually

@kseniyashaydurova 
Done
I had to remove the port validation because it contradicts the idea of using a different number of them.